app/interactors/projects/destroy.rb

```diff
@@ -0,0 +1,19 @@
+module Projects
+  class Destroy
+    include Interactor
+
+    delegate :project_params, to: :context
+
+    def call
+      context.destroy
```

Comment:
What it means?

Reply:
what exactly?

Reply:
Why you call `context.destroy` here? What you want to do?